charset/charset.go

```diff
@@ -56,17 +56,19 @@ var charsetInfos = []*Charset{
 	{CharsetUTF8, CollationUTF8, make(map[string]*Collation), "UTF-8 Unicode", 3},
 	{CharsetUTF8MB4, CollationUTF8MB4, make(map[string]*Collation), "UTF-8 Unicode", 4},
 	{CharsetASCII, CollationASCII, make(map[string]*Collation), "US ASCII", 1},
-	{CharsetLatin1, CollationLatin1, make(map[string]*Collation), "Latin1", 1},
-	{CharsetBin, CollationBin, make(map[string]*Collation), "binary", 1},
+	{CharsetLatin1, CollationLatin1, make(map[string]*Collation), "cp1252 West European", 1},
+	{CharsetBin, CollationBin, make(map[string]*Collation), "Binary pseudo charset", 1},
+	{CharsetGBK, CollationGBKChineseCi, make(map[string]*Collation), "GBK Simplified Chinese", 1},
```

Comment:
https://en.wikipedia.org/wiki/GBK_(character_encoding)
```suggestion
	{CharsetGBK, CollationGBKChineseCi, make(map[string]*Collation), "GBK Simplified Chinese", 2},
```

> A character is encoded as 1 or 2 bytes.